modules/nw-control-dns-records-public-hosted-zone-azure.adoc

```diff
@@ -55,15 +55,16 @@ openshift-console          console             console-openshift-console.apps.te
 openshift-console          downloads           downloads-openshift-console.apps.test.azure.example.com                     downloads           http    edge/Redirect          None
 ----
 
-. Get the list of dns zones to find the one which corresponds to the previously found route's domain:
+. Get the list of DNS zones by running the following command:
 +
 [source,terminal]
 ----
 $ az network dns zone list --resource-group "${RESOURCE_GROUP}"
 ----
 
-. Create `ExternalDNS` resource for `route` source:
+. Create a YAML file that defines the `ExternalDNS` object:
```

Comment:
We might want to name the YAML file explicitly in the procedure text if it will be used later. It's easy to accidentally miss.